modules/javafx.web/src/main/java/com/sun/javafx/webkit/prism/WCGraphicsPrismContext.java

```diff
@@ -700,6 +701,23 @@ public void setMiterLimit(float miterLimit) {
         state.getStrokeNoClone().setMiterLimit(miterLimit);
     }
 
+    @Override
+    public boolean strokeContains(WCPath path, float x, float y) {
+        BasicStroke stroke = state.getStrokeNoClone().getPlatformStroke();
+        boolean result = stroke
+            .createCenteredStrokedShape((Path2D) path.getPlatformPath())
+            .contains(x, y);
+
+        if (log.isLoggable(Level.FINE)) {
```

Comment:
Log must be added as a first statement of the method. Refer other methods.

Reply:
I repaired the logging call (it was the wrong syntax), but left it at the current place. My reasoning: the other methods in `WCGraphicsPrismContext` are mostly `void` methods. For these logging is only relevant to get the input parameters. For `strokeContains` the return value is very interesting and I consider it more valuable, than being in line with methods, that are a different category.